lib/generators/decidim/templates/app.json.erb

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "<%= app_name %>",
+  "description": "TODO: Add a short description about <%= app_name %>",
+  "keywords": [],
+  "addons": [
+    "heroku-postgresql:hobby-dev",
+    "heroku-redis:hobby-dev",
+    "newrelic:wayne",
+    "papertrail:choklad",
+    "sendgrid:starter",
+    "memcachier:developer",
+    "rollbar:free"
+  ],
+  "env": {}
```

Comment:
We should put at least some environment variables like `SECRET_KEY_BASE`: https://github.com/AjuntamentdeBarcelona/decidim.barcelona/blob/master/app.json#L14